docs/reference/errors-and-warnings/NU3029.md

```diff
@@ -0,0 +1,24 @@
+---
+title: NuGet Warning NU3029
+description: NU3029 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 3/18/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3029
+---
+
+# NuGet Warning NU3029
+
+<pre>Package 'SamplePackage v1.0.0' from source 'http://contoso.com/index.json': The timestamp signature is invalid.</pre>
+
```

Comment:
in all the cases above, we were specific about the problem. Why is this an exception? Why do we not talk about what about the signature makes it invalid?